students/urtnowski_daniel/lesson_10_organizing_files/filling_in_the_gaps.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+"""
+filling_in_the_gaps.py: a practice project "Filling in the Gaps" from:
+https://automatetheboringstuff.com/chapter9/
+
+The program finds in a single folder all files named with a given prefix
+followed by a number and corrects any gaps in the numbering
+
+Usage: ./filling_in_the_gaps.py <prefix> <folder_path>
+
+"""
+
+__author__ = "Daniel Urtnowski"
+__version__ = "0.1"
+
+
+import sys
+import os
+import re
+import shutil
+
+
+def get_args():
+
+    args_cnt = len(sys.argv)
+
+    if args_cnt != 3:
+        print('Usage: ./filling_in_the_gaps.py <prefix> <folder_path>')
+        return None
+
+    else:
```

Comment:
`else` jest zbędny, jeżeli program dochodzi do tego etapu to znaczy, że `args_cnt` nie jest spełnione, bo jeżeli by było, to by zadziałał `return`, więc można od razu:
```
if args_cnt != 3:
    print('Usage: ./filling_in_the_gaps.py <prefix> <folder_path>')
    return

prefix, path = sys.argv[1:]
...
```